src/services/sign-json-rpc-request.ts

```diff
@@ -117,7 +114,7 @@ export class SignJsonRpcRequest {
       ? await getRpcProvider(provider)
       : getDefaultNetwork();
 
-    const path = wallet.path?.replace(TRON_COIN_TYPE, ETH_COIN_TYPE)!;
+    const path = Wallet.getById(wallet.address)?.getPath(provider)!;
```

Comment:
Do we need getById?
wallet is IWalletModel so you can call directly wallet?.getPath(provider)!